keyvi/include/keyvi/dictionary/completion/prefix_completion.h

```diff
@@ -204,10 +212,17 @@ final {
                     continue;
                   }
 
+                  if (data->traverser.GetDepth() >= query_length  + max_edit_distance) {
+                    data->traverser.Prune();
+                    data->traverser++;
+                    continue;
+                  }
```

Comment:
I do not understand that one.  

It seems to me that you really want fuzzy matching but this method provides Fuzzy**completion**

so if you want fuzzy matching, we should implement fuzzy matching.

Reply:
Yeah, true. Will create a new method called Fuzzy**match**.

Reply:
I was thinking about implementing Fuzzy**match** method, and the thing is I don't wanna diverge from existing interface much. But on the other hand it feels like another version of class `PrefixCompletion` will be just redundant, and we can just have dedicated function for it in a separate namespace. 
What are your thoughts about this ?

Reply:
What about a method as part of dictionary? There is already a GetNear() method which is sort of a fuzzy match as well. 

Dictionary::GetFuzzy(...) ??

Reply:
Yeah, that sounds good!